webapp/src/Controller/API/JudgehostController.php

```diff
@@ -200,11 +200,15 @@ public function createJudgehostAction(Request $request) : array
         /** @var Judging[] $judgings */
         $judgings = $this->em->createQueryBuilder()
             ->from(Judging::class, 'j')
-            ->innerJoin('j.judgehost', 'jh')
             ->leftJoin('j.rejudging', 'r')
+            ->leftJoin('j.runs', 'jr')
+            ->leftJoin('jr.judgetask', 'jt')
```

Comment:
Since we do an inner join on judgehost below these could also be inner right? Left is also fine, just mentioning it

Reply:
> We should still think about what to do with judging.judgehost. Do you also think it should be dropped in general?

That is #1054?

Reply:
Done